dropgz/build/windows.Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN tdnf install -y unzip
 RUN tdnf upgrade -y && tdnf install -y ca-certificates
 
 FROM tar AS azure-vnet
-ARG AZCNI_VERSION=v1.5.28
+ARG AZCNI_VERSION=v1.6.0
```

Comment:
With this do we need to make a `release/v1.5` branch now @rbtr?

Reply:
Yes

Reply:
I'm making a release right after the merging the PR. If you need to make a 1.5 release then let me know If I need to remove this tag?

Reply:
Changed it back to 5.28. Will make another PR for 1.6 dropGZ